test/integration/shared/lockup-linear/LockupLinear.t.sol

```diff
@@ -60,60 +60,60 @@ abstract contract LockupLinear_Integration_Shared_Test is Lockup_Integration_Sha
 
     /// @dev Creates the default stream that is not cancelable.
     function createDefaultStreamNotCancelable() internal override returns (uint256 streamId) {
-        LockupLinear.CreateWithRange memory params = _params.createWithRange;
+        LockupLinear.CreateWithTimestamps memory params = _params.createWithTimestamps;
         params.cancelable = false;
-        streamId = lockupLinear.createWithRange(params);
+        streamId = lockupLinear.createWithTimestamps(params);
     }
 
     /// @dev Creates the default stream with the NFT transfer disabled.
     function createDefaultStreamNotTransferable() internal override returns (uint256 streamId) {
-        LockupLinear.CreateWithRange memory params = _params.createWithRange;
+        LockupLinear.CreateWithTimestamps memory params = _params.createWithTimestamps;
         params.transferable = false;
-        streamId = lockupLinear.createWithRange(params);
+        streamId = lockupLinear.createWithTimestamps(params);
     }
 
     /// @dev Creates the default stream with the provided end time.
     function createDefaultStreamWithEndTime(uint40 endTime) internal override returns (uint256 streamId) {
-        LockupLinear.CreateWithRange memory params = _params.createWithRange;
+        LockupLinear.CreateWithTimestamps memory params = _params.createWithTimestamps;
         params.range.end = endTime;
-        streamId = lockupLinear.createWithRange(params);
+        streamId = lockupLinear.createWithTimestamps(params);
     }
 
-    /// @dev Creates the default stream with the provided createWithRange.
-    function createDefaultStreamWithRange(LockupLinear.Range memory createWithRange)
+    /// @dev Creates the default stream with the provided createWithTimestamps.
+    function createDefaultStreamWithRange(LockupLinear.Range memory createWithTimestamps)
```

Comment:
This appears to be inconsistent with the datatype used here.

Instead of `LockupLinear.Range memory createWithTimestamps`, how about `LockupLinear.Range memory range`?

On line 88, we can say `params.range = range`.

Reply:
Yeah, this is indeed a mistake. And it was a mistake even before this PR. Good catch.